scrapers/Mylf.yml

```
@@ -33,9 +33,18 @@ xPathScrapers:
                 with: "hi.jpg"
       Studio:
         Name:
-          selector: //div[contains(@class,"siteName")]/a[@href]/text()
+          selector: //div[contains(@class,"siteName")]/a[@href]/text()[2]
```

We can use
```suggestion
          selector: //div[contains(@class,"siteName")]/a[@href]
```
and let stash parse the inner text....